common/persistence/dataInterfaces.go

```diff
@@ -1449,6 +1449,37 @@ type (
 		persistenceblobs.ImmutableClusterMetadata
 	}
 
+	// GetActiveClusterMembersRequest is the response to GetActiveClusterMembers
+	GetActiveClusterMembersRequest struct {
+		LastHeartbeatWithin time.Duration
```

Comment:
I thought we already have expiry, what's the point of taking in this parameter?

Reply:
We need a way to list all members including timed out ones. 
So I would rename this method to GetClusterMembers(LastHeartbeatWithin)

Reply:
Discussed in person that expiry is for long term expiration out of table (say 48 hours), but services will apply a much shorter expiration (say 10 minutes) from the read side on LastHeartbeat.